include/mrdox/Metadata/Symbols.hpp

```diff
@@ -14,6 +14,7 @@
 
 #include <mrdox/Platform.hpp>
 #include <llvm/ADT/StringRef.h>
+#include <llvm/ADT/StringExtras.h>
```

Comment:
well don't be adding stuff from llvm you need to be removing it !

Reply:
Oops, forgot to remove that